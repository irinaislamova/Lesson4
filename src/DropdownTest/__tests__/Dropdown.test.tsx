/**
 * @jest-environment jsdom
 */


import React from 'react';
import { shallow, configure } from 'enzyme';
import { DropdownTest } from '../DropdownTest';
import Adapter from 'enzyme-adapter-react-16';

configure({adapter: new Adapter()});

describe('DropdownTest', () => {
	test('should render', () => {
		const wrapper = shallow(<DropdownTest  
			children={<div />}
			button={<button />}
			/>)
		console.log(window);
		expect(wrapper).toBeDefined();
		expect(wrapper.find('div.container').isEmptyRender()).toBeFalsy();
	})
	test('should render (snapshot)', () => {
		const wrapper = shallow(<DropdownTest 
			children={<div />}
			button={<button />}
			/>)
		expect(wrapper).toMatchSnapshot();
	})
})
