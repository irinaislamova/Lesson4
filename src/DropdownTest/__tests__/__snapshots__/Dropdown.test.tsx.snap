// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DropdownTest should render (snapshot) 1`] = `
<div>
  <div
    className="container"
    onClick={[Function]}
  >
    <button />
  </div>
</div>
`;
